components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/importexport/utils/APIExportUtil.java

```diff
@@ -115,6 +115,7 @@ public static void retrieveApiToExport(String archiveBasePath, API apiToReturn,
 
             //export thumbnail
             exportAPIThumbnail(archivePath, apiIDToReturn, registry);
+            exportSOAPToRESTMediation(archivePath,apiIDToReturn, registry);
```

Comment:
formatting issue ( space needed after the comma). Keep a constant format

Reply:
resolved by #8332